Component/Mapping/CMakeLists.txt

```diff
@@ -0,0 +1,145 @@
+cmake_minimum_required(VERSION 3.12)
+project(Sofa.Component.Mapping LANGUAGES CXX)
+
+set(SOFACOMPONENTMAPPING_SOURCE_DIR "src/sofa/component/mapping")
+
+set(HEADER_FILES
+    ${SOFACOMPONENTMAPPING_SOURCE_DIR}/config.h.in
+    ${SOFACOMPONENTMAPPING_SOURCE_DIR}/init.h
+    ${SOFACOMPONENTMAPPING_SOURCE_DIR}/BarycentricMapping.h
```

Comment:
I am used to write the files in alphabetic order in a CMakeLists.txt.

Reply:
For these 2 files in particular, I prefer to put them at the beginning, as there are not related to components (just to setup)